src/alto-ui/Form/CheckBox/CheckBox.js

```diff
@@ -32,6 +32,7 @@ CheckBox.propTypes = {
   disabled: PropTypes.bool,
   hideLabel: PropTypes.bool,
   readOnly: PropTypes.bool,
+  defaultChecked: PropTypes.bool,
```

Comment:
you are spreading props so not needed. You should have an eslint warning btw.

Reply:
Ok, fixed